playbooks/support.md

```diff
@@ -143,6 +146,21 @@ You can find the point people in the [Potential Wiki](https://github.com/artsy/p
 - [Domain Models](https://github.com/artsy/potential/blob/master/platform/DomainModels.md)
 - [Status Page](status.artsy.net)
 
+## Expectations (cheat sheet)
+
+### What is expected of you
+- _During work hours_, respond to #incidents in a timely manner.
+- Address all scheduling issues/conflicts for your shift as early as you know about them.
+- Communicate to your manager when something comes up that prevents you from being on-call.
+- Lean on the entire team for help-- no one expects you to know how to debug every system.
+- Improve the process where you can (including educating others).
+- Help facilitate solutions, if need-be.
+
+### What is not expected of you
+- To know the ins and outs of every system.
+- To wake up at night for a non-emergency.
+- To fix all of the bugs you find.
```

Comment:
This is good stuff but is basically the short version of the longer list above. Ironically, this makes the doc _longer_ and harder to follow, because now there are 2 separate lists of what to do. Would be even better to integrate these above (maybe in bold for clarity) and shorten the rest.

Reply:
I think that in a case like this, where people may be anxious about their responsibilities, a bit longer and explicit in what they do _not_ need to worry about can be helpful.